components/match2/commons/match_group_util.lua

```diff
@@ -328,17 +328,19 @@ Returns a match struct for use in a bracket display or match summary popup. The
 bracket display and match summary popup expects that the finals match also
 include results from the bracket reset match.
 ]]
-function MatchGroupUtil.fetchMatchForBracketDisplay(bracketId, matchId)
+function MatchGroupUtil.fetchMatchForBracketDisplay(bracketId, matchId, config)
+	config = config or {}
 	local bracket = MatchGroupUtil.fetchMatchGroup(bracketId)
 	local match = bracket.matchesById[matchId]
 
 	local bracketResetMatch = match
 		and match.bracketData.bracketResetMatchId
 		and bracket.matchesById[match.bracketData.bracketResetMatchId]
-	if bracketResetMatch then
+
+	if bracketResetMatch and not config.seperateBracketResetMatch then
 		return MatchGroupUtil.mergeBracketResetMatch(match, bracketResetMatch)
 	else
-		return match
+		return match, bracketResetMatch
```

Comment:
How about?
```lua
	if bracketResetMatch and config.mergeBracketReset then
		return MatchGroupUtil.mergeBracketResetMatch(match, bracketResetMatch)
	else
		return match, bracketResetMatch
```
(with `mergeBracketReset` defaulting true=

Reply:
do not like that:
- would nee changing 15 callers
- would mean to not ever call this function if you want them seperated, would just call the following directly:
```lua
local bracket = MatchGroupUtil.fetchMatchGroup(bracketId)
local match = bracket.matchesById[matchId]
local bracketResetMatch = match
	and match.bracketData.bracketResetMatchId
	and bracket.matchesById[match.bracketData.bracketResetMatchId]
```
(you need the `MatchGroupUtil.fetchMatchGroup` call for the reset match anyways, so it would be a 1:1 line replacement for the match  and for the reset match you would need to add those lines anyways)

let me know if you would rather have those 5 lines in the sc/sc matchsummary or this pr^^

Reply:
> would need changing 15 callers

I don't why?

Reply:
> > would need changing 15 callers
> 
> I don't see why?

ah didn't see the
> (with mergeBracketResetMatch defaulting true)